src/loader.js

```diff
@@ -75,13 +75,13 @@ async function load(source) {
     let previousDocument = global.document;
     let previousElement = global.Element;
     // https://github.com/pmmmwh/react-refresh-webpack-plugin/issues/176#issuecomment-683150213
-    global.$RefreshReg$ = noop;
-    global.$RefreshSig$ = () => noop;
+    global.$RefreshReg$ = previousRefreshReg || noop;
```

Comment:
I'd love to use 
```suggestion
    global.$RefreshReg$ ??= noop;
```

here, but it isn't supported until Node@15, and feels too early to introduce here.

Reply:
I agree